src/java.base/windows/native/libnio/MappedMemoryUtils.c

```diff
@@ -76,7 +76,7 @@ Java_java_nio_MappedMemoryUtils_force0(JNIEnv *env, jobject obj, jobject fdo,
         if ((result != 0) || (GetLastError() != ERROR_LOCK_VIOLATION))
             break;
         retry++;
-    } while (retry < 3);
+    } while (retry < 5);
```

Comment:
This retry is a workaround for what we think is a bug in Windows. I remember there was speculation that it was something to do with the NTFS transaction log being full. It would be useful if get a comment from someone from Microsoft on this.

Reply:
A full transaction log was indeed a speculation logged in the issue's comments.